src/main/js/SaveApi.js

```diff
@@ -6,22 +6,23 @@ import { Fetch, getRestUrl, sseService, loadingIndicator, capabilityAugmenter }
 export class SaveApi {
 
     indexRepo(organization, teamName, repoName) {
-        const createUrl = `${getRestUrl({organization})}/pipelines/`;
-
+        const createUrl = repoName ? `${getRestUrl({organization})}pipelines/` : `${getRestUrl({organization, pipeline: teamName})}`;
```

Comment:
This is not needed. What this is doing is adding repo/trigger indexing to org folder. Git and Bitbucket are not org folder, they create multi-branch pipeline. To trigger indexing There is already code I added, see https://github.com/jenkinsci/blueocean-pipeline-editor-plugin/blob/master/src/main/js/EditorPage.jsx#L411. There is no changes needed in SaveApi. 🐛